recipes/gottcha/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: gottcha
+  version: "1.0"
+
+source:
+  git_url: https://github.com/LANL-Bioinformatics/GOTTCHA.git
```

Comment:
can you pin please a release? This will take always the latest master. Ideally a tarball would be great.